app/serializers/api/v2/npq_application_serializer.rb

```diff
@@ -4,60 +4,9 @@
 
 module Api
   module V2
-    class NPQApplicationSerializer
```

Comment:
I wouldn't worry about V2 it's not released and fallen so far behind at the moment